phalcon/mvc/model/manager.zep

```diff
@@ -1715,18 +1715,9 @@ class Manager implements ManagerInterface, InjectionAwareInterface, EventsAwareI
 	 */
 	public function executeQuery(string! phql, var placeholders = null, var types = null) -> <QueryInterface>
 	{
-		var dependencyInjector, query;
-
-		let dependencyInjector = <DiInterface> this->_dependencyInjector;
-		if typeof dependencyInjector != "object" {
-			throw new Exception("A dependency injection object is required to access ORM services");
-		}
+		var query;
 
-		/**
-		 * Gets Query instance from DI container
-		 */
-		let query = <QueryInterface> dependencyInjector->get("Phalcon\\Mvc\\Model\\Query", [phql, dependencyInjector]);
-		let this->_lastQuery = query;
+		let query = this->createQuery(phql);
```

Comment:
It's on line [1709](https://github.com/phalcon/cphalcon/pull/12297/files#diff-8212f6f15cd92872f55e7d0e66172601L1709)